packages/background/src/controllers/GasPricesController.ts

```diff
@@ -203,6 +204,26 @@ export class GasPricesController extends BaseController<GasPricesControllerState
         return this.getState(chainId).gasPricesLevels;
     }
 
+    /**
+     * Get suggested high price in GWEI
+     */
+    public getHighGasPriceInGwei(chainId?: number): string {
+        const { estimatedBaseFee, gasPricesLevels } = this.getState(chainId);
+        const fast = gasPricesLevels.fast;
+        // If we have gasPrice, it means it's a non EIP-1559 compatible network so we return it as it is.
+        if (fast.gasPrice) {
+            return formatUnits(fast.gasPrice, 'gwei');
+        } else {
+            // For EIP-1559 networks, we return baseFee + high wTip
+            let totalGwei = BigNumber.from(0);
+            if (estimatedBaseFee && fast.maxPriorityFeePerGas) {
```

Comment:
if this is false, totalGwei will be zero. is this ok?

Reply:
Both variables are nullable because we use the same object for non-EIP-1159 networks, so those properties won't coexist with values at the same time. The conditional is only meant to prevent using the **!** operator.

In any case, if the totalGwei returns 0, the quote request to the API will fail and it'll display a message. But that would mean that we're not being able to calculate gas prices for any operation within the wallet.